tests/service/data/ToolsVersion12.fsproj

```diff
@@ -41,13 +41,6 @@
   <ItemGroup>
     <Compile Include="main.fs" />
   </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Test\test.fsproj">
-      <Name>Interfaces</Name>
-      <Project>{00000000-0000-0000-0000-000000000001}</Project>
-      <Private>True</Private>
-    </ProjectReference>
-  </ItemGroup>
```

Comment:
This file was invalid (the referenced project doesn't exist), so the test was failing for a reason unrelated to what it should test for.